src/ModelContextProtocol/Protocol/Types/LoggingMessageNotificationParams.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Text.Json;
+
+namespace ModelContextProtocol.Protocol.Types;
+
+/// <summary>
+/// Sent from the server as the payload of "notifications/message" notifications whenever a log message is generated.
+/// 
+/// If no logging/setLevel request has been sent from the client, the server MAY decide which messages to send automatically.
+/// <see href="https://github.com/modelcontextprotocol/specification/blob/main/schema/2024-11-05/schema.json">See the schema for details</see>
+/// </summary>
+public class LoggingMessageNotificationParams
+{
+    /// <summary>
+    /// The severity of this log message.
+    /// </summary>
+    [System.Text.Json.Serialization.JsonPropertyName("level")]
```

Comment:
Nit: using for System.Text.Json.Serialization and then remove from here and below